src/hash_functions.cpp

```diff
@@ -31,4 +31,21 @@ std::size_t CPublicCoinHash::operator ()(const sigma::PublicCoin& coin) const no
     return result;
 }
 
+
+CMintedCoinInfo CMintedCoinInfo::make(CoinDenomination denomination,  int coinGroupId, int nHeight) {
```

Comment:
Probably best to change filename as these are the only objects present and are unrelated to hash functions